<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QDialog" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Реєстрація</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Прізвище, ім'я:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Ел. пошта:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ІПН:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Номер телефону:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Пароль:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="line_name">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Прізвище</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_email">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>email</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_ipn">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ІПН</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_number">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>123</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_password">
         <property name="font">
          <font>
           <family>Alef</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>123</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_reg">
       <property name="font">
        <font>
         <family>Alef</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Зареєструватися</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_reject">
       <property name="font">
        <font>
         <family>Alef</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Назад</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_reject</sender>
   <signal>clicked()</signal>
   <receiver>Registration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
